mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -552,6 +552,40 @@ public void testPartitionedWrite() throws IOException {
     HiveIcebergTestUtils.validateData(table, records, 0);
   }
 
+  @Test
+  public void testIdentityPartitionedWrite() throws IOException {
```

Comment:
Can we add another test or extend this one with a scenario where the table is partitioned by more than one identity column?

Reply:
Done